REST/aq/index.php

```diff
@@ -21,31 +21,65 @@
 
 $f3->set('AUTOLOAD', 'sql/;view/;controller/;model/;py/;');
 
-$f3->route('GET /v1/read/@keyword', function($f3){
+$f3->route('GET /v1/read/@keyword', function($f3) {

```

Comment:
The route format here is not what is passed by the gateway. See [API Url Syntax](https://github.com/uw-thalesians/perceptia-servers/tree/develop/api#url-syntax) for specifics. The gateway will proxy all requests to "/api/vX/anyquiz/*" where X is any integer greater than or equal to 0 (regex {v[0-9]+}, such as "v1" or "v12". Just want to make sure your code either rewrites the path to the format you expect or is handling this in some other way.